server/app.py

```diff
@@ -3,17 +3,46 @@
 The endpoint called `endpoints` will return all available endpoints.
 """
 
-from flask import Flask, send_from_directory
+import os
+import pathlib
+from flask import Flask, send_from_directory, redirect, request
 from flask_restful import Api, Resource
 from .src.endpoints.endpoints import Endpoints, HelloWorld
+from google_auth_oauthlib.flow import Flow
+
 
 app = Flask(__name__, static_url_path='', static_folder='frontend/build')
 api = Api(app)
 
+
+GOOGLE_CLIENT_ID_TEST = "497541279341-qtudp4uvo0g39s0o4ops0mr2dsvemnp5.apps.googleusercontent.com"
+CLIENT_SECRET_FILE = os.path.join(pathlib.Path(__file__).parent, "client_secret_test.json")
+flow = Flow.from_client_secrets_file(
+    client_secrets_file=CLIENT_SECRET_FILE,
+    scopes=["https://www.googleapis.com/auth/userinfo.profile", "https://www.googleapis.com/auth/userinfo.email", "openid"],
```

Comment:
Same with this. We can have these constants and variables in the endpoint's file.